vulcanai2/datasets/tabulardataset.py

```diff
@@ -290,37 +289,52 @@ def create_one_hot_encoding(self, column_name, prefix_sep="@"):
         """
 
         # TODO: ensure dummy_na =False is what you want
-        if column_name in list(self.df):
-            self.df = pd.get_dummies(self.df, columns=[column_name],
-                                     prefix_sep=prefix_sep)
-            logger.info(f"Successfully encoded {column_name}")
+        self.df = pd.get_dummies(self.df, columns=[column_name],
+                                    prefix_sep=prefix_sep)
+        logger.info(f"Successfully encoded {column_name}")
 
-        else:
-            logger.info(f"Col {column_name} does not exist")
-
-    def reverse_create_all_one_hot_encodings(self, prefix_sep="@"):
+    # if a use case presents itself column_name could easily become a list
+    def reverse_create_one_hot_encoding(self, column_name=None,
+                                        reverse_all=False, prefix_sep="@"):
         """
         Undo the creation of one-hot encodings, if prefix_sep was used
-        to create one-hot encodings and nowhere else.
-
+        to create one-hot encodings and nowhere else. If a column_name is
+        provided, only that column will be reverse-encoded, even if reverse_all
+        is set to true.
```

Comment:
@rfratila let me know if you think it's overkill to have both reverse_all and column_name set to a negative value, so that the user has to provide one or the other

Reply:
tis a wee bit weird

Reply:
yeah let's just have `column_name` and if it's none it will reverse all, otherwise just the ones specified

Reply:
cool